cli/src/lib/generateManifests.js

```diff
@@ -0,0 +1,151 @@
+const { reporter, chalk } = require('@dhis2/cli-helpers-engine')
```

Comment:
This file is renamed from 'generateManifest.js', and adds a `manifest.json` in addition to `manifest.webapp`